packages/components/src/modal/index.tsx

```diff
@@ -365,6 +365,7 @@ function UnforwardedModal(
 									? focusOnMountRef
 									: null,
 							] ) }
+							className="components-modal__children-container"
```

Comment:
It's a bit challenging to do a 100% full screen modal where the content's height matches the modal's height. This additional class allows us to do this. I know it's not great but it's not worse than how we've been using modal before.

Reply:
I had made a similar [change](https://github.com/WordPress/gutenberg/pull/50099/files#diff-7d9147900701280de3fb09897ba9b4e36f424b18412332fc2f72774ef1afe035R255) a few years back and it was [reverted](https://github.com/WordPress/gutenberg/pull/50655) (this specific change) from components team. The PR adjusted the styles of the component I wanted to have full height.